src/constants/Style.js

```diff
@@ -1,4 +1,10 @@
 module.exports = {
+  BreakPoints: {
+    large: 1200,
+    medium: 750,
+    small: 320
```

Comment:
I started down that path but then noticed the `Column` component merges a `breakpoints props with this `BreakPoint` object to keep the `Column` component dynamic.  Changing the shape of that prop to match the up case here would be a breaking change so I thought I would leave it lower case for now.

I'm assuming that `Object.assign` wouldn't merge two keys if one was upper case and the other lower case.  If I'm wrong on that assumption please let me know.